tensorflow/contrib/periodic_resample/python/kernel_tests/periodic_resample_op_test.py

```diff
@@ -103,6 +106,18 @@ def testPeriodicResampleErrors(self):
           '4, to be the same as the length of the desired shape, 3'):
         periodic_resample(input_tensor, [None, 4, 4]).eval()
 
+  def testPeriodicResampleGradient(self):
+    desired_shape = numpy.array([4, 4, None])
+    result_shape = (4, 4, 1)
+    input_shape = (2, 2, 4)
+    with self.test_session() as sess:
+      variables.global_variables_initializer().run()
```

Comment:
Your test does not contain any variables, so I'm not sure why this is needed. (The same goes for the existing test.)

Reply:
done